compose/ChartboostMediationDemoCompose/src/main/java/com/chartboost/mediation/sdk/demo/AdController.kt

```diff
@@ -554,7 +321,7 @@ object AdController : DefaultLifecycleObserver {
             }
 
             logState.add("Fullscreen ad is about to show")
-            val result = fullscreenAd?.show(context)
+            val result = fullscreenAd?.show(context as Activity)
```

Comment:
In Kotlin this is not a safe cast. https://kotlinlang.org/docs/typecasts.html#safe-nullable-cast-operator

Reply:
https://github.com/ChartBoost/chartboost-mediation-android-sdk-demo/pull/39/commits/fbde09ae84f9524f054bdb4485c85ead12cf66d6